src/opm/parser/eclipse/EclipseState/Aquancon.cpp

```diff
@@ -113,10 +124,60 @@ namespace Opm {
     }
 
 
+    bool Aquancon::cellActive(const Opm::EclipseGrid& grid, const int i, const int j, const int k)
+    {
+        if ( i <= 0 || j <=0 || k <=0
+            || size_t(i) > grid.getNX()
+            || size_t(j) > grid.getNY()
+            || size_t(k) > grid.getNZ())
+        {
+            return false;
+        }
+
+        return grid.cellActive(i-1 , j-1, k-1 );
+    }
+
+    bool Aquancon::neighborCellDirectionActive(const Opm::EclipseGrid& grid,
+           const int i, const int j, const int k, const Opm::FaceDir::DirEnum faceDir)
+    {
+        int iNeighbor = i;
+        int jNeighbor = j;
+        int kNeighbor = k;
+
+        switch(faceDir) {
```

Comment:
This can be rewritten with direct return:
```
switch (faceDir) {
    case FaceDir::XMinus:
           return cellActive(grid, i - 1, j, k);
    case FaceDir::XPlus:
           return cellActive(grid, i+1, j, k);
....
```